vespalib/src/vespa/vespalib/util/hdr_abort.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#pragma once
+
+namespace vespalib {
+
+extern void hdr_abort(const char *message,
+                      const char *file,
+                      unsigned int line) __attribute__((__noreturn__));
```

Comment:
Should be possible to use standardized `[[noreturn]]` attribute here instead of GCC attribute (https://en.cppreference.com/w/cpp/language/attributes)